smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -931,23 +932,31 @@ private handlePoll(body) {
 	if (state.updating) {
 		// If user just executed commands, then ignore poll to not confuse the turning on/off state
 		return []
-                }
+	}
 
 	def bulbs = getChildDevices()
 	for (bulb in body) {
 		def device = bulbs.find{it.deviceNetworkId == "${app.id}/${bulb.key}"}
 		if (device) {
 			if (bulb.value.state?.reachable) {
+				if (state.bulbs[bulb.key]?.online == false) {
+					// light just came back online, notify device watch
+					def lastActivity = now()
+					sendEvent(name: "deviceWatch-lastActivity", value: lastActivity, description: "Last Activity is on ${new Date((long) lastActivity)}", displayed: false, isStateChange: true)
```

Comment:
I would prefer if we found another name and value for this event. This combination is something we chose to signal that a device/dth was pinged by device-watch when it actually did have activity inside the reporting interval. So part of the workaround for the isStateChange events on Kafka problem.